Cargo.lock

```diff
@@ -1714,6 +1714,7 @@ dependencies = [
  "common-treenode",
  "daft-core",
  "daft-sketch",
+ "indexmap 2.2.6",
```

Comment:
not blocking but we should look into why we have multiple versions of indexmap

Reply:
It looks like it's because of arrow2's dependency on indexmap 1.6.